src/Raven.Server/Rachis/RachisConsensus.cs

```diff
@@ -1879,7 +1911,25 @@ public void UpdateNodeTag(TransactionOperationContext context, string newTag)
                 var state = context.Transaction.InnerTransaction.CreateTree(GlobalStateSlice);
                 state.Add(TagSlice, str);
             }
+        }
+
+        public void SetSnapshotRequest(TransactionOperationContext context, bool request)
+        {
+            using (Slice.From(context.Transaction.InnerTransaction.Allocator, request.ToString(), out Slice str))
```

Comment:
Why are you storing it as a string? Can just store a byte.

Reply:
done.